examples_test.go

```diff
@@ -191,16 +192,12 @@ func TestExampleToString_external(t *testing.T) {
 	ExampleToString_external()
 }
 
-// Demonstrates string formatting of wrapped errors that originate from global root errors (created
-// via eris.NewGlobal). You can try this example in the Go playground
-// (https://play.golang.org/p/8YgyDwk9xBJ).
+// Demonstrates string formatting of wrapped errors that originate from global root errors. You can
+// try this example in the Go playground (https://play.golang.org/p/8YgyDwk9xBJ).
```

Comment:
are you planning to update the example links later?

Reply:
yeah, can't update them until after release